script/vnctest

```diff
@@ -19,35 +19,43 @@ sub usage ($r = 0) {
     example: --hostname localhost --port 590x';
     exit $r;
 }
-my %options;
-GetOptions(\%options, 'hostname=s', 'port=s', 'password=s', 'update-delay=s', 'verbose|v', 'help|h|?') or usage(1);
-usage if $options{help};
-
-my $update_delay = $options{'update-delay'} // 1;
-my $log = Mojo::Log->new;
-$log->level($options{verbose} ? 'debug' : 'info');
-$log->debug('Loading tinycv');
-cv::init;
-require tinycv;
-
-my $image_path = '/tmp/vnc-framebuffer.png' // $ENV{VNC_TEST_TEMP_IMAGE_PATH};
-unlink $image_path;
-exec "$Bin/debugviewer/debugviewer", $image_path if $ENV{VNC_TEST_DEBUGVIEWER} && fork == 0;
-
-$log->info('Initializing VNC');
-my $vnc = consoles::VNC->new(%options);
-my $incremental = 0;
-$vnc->login;
-
-while(1) {
-    $log->info('Send update request');
-    $vnc->send_update_request($incremental);
-    $incremental = 1;
-    $log->debug('Updating frame buffer');
-    if ($vnc->update_framebuffer) {
-        my $frame_buffer = $vnc->_framebuffer;
-        $log->debug($frame_buffer ? 'Update received, has frame buffer' : 'Update received');
-        $frame_buffer->write($image_path) if $frame_buffer;
+
+sub parse_args () {
+    my %options;
+    GetOptions(\%options, 'hostname=s', 'port=s', 'password=s', 'update-delay=s', 'verbose|v', 'help|h|?') or usage(1);
+    usage if $options{help};
+}
```

Comment:
You missed that line from #2638 
```suggestion
    return \%options;
}
```

Reply:
I didnt. I intentionally removed it. it was uncovered. then I tried to run `vnctest --help` and there was output. So I think it is not needed. I will take another look later.

Reply:
That can't work. options is for sure used

Reply:
And we can actually cover that line as well by calling `main(parse_args())` instead of `main($hashref)` directly. You would set `local @ARGV = qw(--verbose)` for that to simulate the command line parameters.

Reply:
I just saw that. i will check